src/SampleLibraryTests/SampleLibraryTests.csproj

```diff
@@ -102,31 +97,43 @@
     </Reference>
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
-    <Reference Include="ProtoCore, Version=2.8.0.2045, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\DynamoVisualProgramming.Core.2.7.0.9206\lib\net48\ProtoCore.dll</HintPath>
+    <Reference Include="ProtoCore, Version=2.10.0.3435, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\DynamoVisualProgramming.Core.2.10.0.3435\lib\net48\ProtoCore.dll</HintPath>
     </Reference>
-    <Reference Include="ProtoGeometry, Version=2.7.0.8935, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\DynamoVisualProgramming.ZeroTouchLibrary.2.7.0.9206\lib\net48\ProtoGeometry.dll</HintPath>
+    <Reference Include="ProtoGeometry, Version=2.9.0.3355, Culture=neutral, processorArchitecture=AMD64">

```

Comment:
this is on me I guess - we'll change it during next release cycle - we should be more consistent I agree - but I always wonder if LibG should be versioned according to semantic versioning and not just in lock step with Dynamo. Chalk it up to a known issue.